tests/test_intervals.py

```diff
@@ -358,3 +363,163 @@ def test_getitem_slice_errors(self):
             recomb_map[-101:]
         with pytest.raises(IndexError):
             recomb_map[:-101]
+
+
+class TestReadHapmap:
```

Comment:
We're only testing fileobjs here - it would be nice to verify that we can use a filename as well. Writing a file to a temporary directory would be a good way to check this.